examples/helloworld/send_text_to_room.py

```diff
@@ -0,0 +1,43 @@
+import asyncio
+from markdown import markdown
+
+from mautrix.client import ClientAPI
+from mautrix.types.event.message import Format, TextMessageEventContent
+
+# The user_id you have in your homeserver
+user_id = "@admin:example.com"
+
+# You homeserver addres
+base_url = "https://example.com"
+
+# The generated token for your user on the homeserver
+token = "syt_123_456"
+
+# Create a client that will be connected to your homeserver
+client = ClientAPI(user_id, base_url=base_url, token=token)
+
+
+async def send_message_to_room(room_id: str, message: str, notice: bool = False) -> None:
+
+    # Defines whether the message is a notice or a text message
+    msgtype = "m.notice" if notice else "m.text"
```

Comment:
Whoops, missed this earlier, but it's probably preferable to use the `MessageType` enum from `mautrix.types`.

```suggestion
async def send_message_to_room(room_id: str, message: str, msgtype: MessageType = MessageType.TEXT) -> None:
```

Reply:
Yes, that seems much better to me